cmd/goat/plc.go

```diff
@@ -494,20 +494,12 @@ func runPLCCalcDID(cctx *cli.Context) error {
 	}
 	op := enum.AsOperation()
 
-	if !op.IsGenesis() {
-		return fmt.Errorf("not a genesis op")
-	}
-
-	if !op.IsSigned() {
-		return fmt.Errorf("genesis op must be signed")
-	}
-
-	didplc, err := op.DID()
+	did, err := op.DID() // errors if op is not a signed genesis op
```

Comment:
NOTE: go-didplc needs to be updated before this is valid, pending merge of https://github.com/did-method-plc/go-didplc/pull/9